httpcore5/src/main/java/org/apache/hc/core5/reactor/IOReactorConfig.java

```diff
@@ -205,31 +205,31 @@ public static Builder copy(final IOReactorConfig config) {
 
     public static class Builder {
 
-        private static int DefaultMaxIoThreadCount = -1;
+        private static int DEF_MAX_IO_THREAD_COUNT = -1;
```

Comment:
The convention says that only final static values are uppercase. This is a variable which can be mutated, hence it should follow general camel case rules: `defaultMaxIOThreadCount`